src/styles/blocks/card.css

```diff
@@ -0,0 +1,92 @@
+* {
+  box-sizing: border-box;
+}
+
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  display: block;
+  margin: 0 auto;
+  height: 134px;
+  width: 160px;
+  margin-bottom: 40px;
+}
+
+.card__title {
+  margin-bottom: 4px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--primary-text-color);
+}
+
+.card__code {
+  margin-bottom: 16px;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--secondary-text-color);
+}
+
+.card__rate {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 24px;
+}
+
+.card__reviews {
+  font-size: 10px;
+  line-height: 14px;
+  text-align: center;
+}
+
+.card__price {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin-bottom: 16px;
+}
+
+.card__price-title {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--secondary-text-color);
+}
+
+.card__price-value {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--primary-text-color);
+}
+
+.card__button {
+  display: flex;
+  align-items: center;
+  justify-content: center;
```

Comment:
flex redundant

Reply:
i don't think that "flex" is redutant, because my button is an "a" attribute, which is inline, so I used flex display to expand it + without it  "align-items" and "justify-content" are useless